astro/src/content/quickstarts/quickstart-php-web.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: PHP
+description: Quickstart integration of PHP web application with FusionAuth
+navcategory: getting-started
+prerequisites: Docker version 20
+section: web
+cta: EmailListCTA
+technology: PHP
+language: PHP
+icon: /img/icons/php.svg
+faIcon: fa-php
+url: https://github.com/FusionAuth/fusionauth-quickstart-php-web
+codeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-php-web/main
+---
+
+import Aside from '/src/components/Aside.astro';
+import LoginAfter from '../../diagrams/quickstarts/login-after.astro';
+import LoginBefore from '../../diagrams/quickstarts/login-before.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
```

Comment:
```suggestion
import {RemoteCode, RemoteValue} from '@fusionauth/astro-components';
```

Using [`<RemoteValue>`](https://github.com/FusionAuth/fusionauth-astro-components#remotevalue) to avoid hard-coding some values